lms/envs/common.py

```diff
@@ -3928,3 +3928,9 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 ##################### SUPPORT URL ############################
 SUPPORT_HOW_TO_UNENROLL_LINK = ''
+
+##################### ADG Settings ############################
+
+# Notification email settings
+NOTIFICATION_FROM_EMAIL = 'info@omnipreneurshipacademy.com'
+NOTIFICATION_EMAIL_SUBJECT = 'ADG Notification'
```

Comment:
It seems we aren't using this setting, So i think it is good to remove if we aren't using this.

Reply:
We have alot of settings here in a common.py that aren't using. If that's still bothering I can remove that?